src/c++/perf_analyzer/genai-pa/genai_pa/constants.py

```diff
@@ -25,3 +25,11 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 LOGGER_NAME: str = "genai-pa"
+
+DEFAULT_HTTP_URL = "localhost:8000"
+DEFAULT_GRPC_URL = "localhost:8001"
+
+
+OPEN_ORCA = "openorca"
+CNN_DAILY_MAIL = "cnn_dailymail"
+DEFAULT_INPUT_DATA_JSON = "./llm_inputs.json"
```

Comment:
I think the `./` is implied by default, and it would be easier to use this var with other path related logic if you removed the `./`
```suggestion
DEFAULT_INPUT_DATA_JSON = "llm_inputs.json"
```